eclass/ros-catkin.eclass

```diff
@@ -163,16 +163,19 @@ ros-catkin_src_prepare() {
 	append-cxxflags '-std=c++11'
 }
 
+ROSPREFIX="/opt/ros/${ROSDISTRO}"
+
 # @FUNCTION: ros-catkin_src_configure_internal
 # @DESCRIPTION:
 # Internal decoration of cmake-utils_src_configure to handle multiple python installs.
 ros-catkin_src_configure_internal() {
 	if [ -n "${CATKIN_DO_PYTHON_MULTIBUILD}" ] ; then
 		local sitedir="$(python_get_sitedir)"
+		local sitedir="${sitedir/\/usr\//}"
 		local mycmakeargs=(
 			"${mycmakeargs[@]}"
 			-DPYTHON_EXECUTABLE="${PYTHON}"
-			-DPYTHON_INSTALL_DIR="${sitedir#${EPREFIX}/usr/}"
+			-DPYTHON_INSTALL_DIR="${sitedir#${EPREFIX}}"
```

Comment:
I does not in this case. There's a slight work around I'm still iterating on that deals with this variable.